### ISSUE TITLE
Advices needed for cifar database

### ISSUE THREAD

**Issue by cvtower** [miaow1988/ShuffleNet_V2_pytorch_caffe#7]
Hi @miaow1988 ,

Nice work!
Would you please check the network config bellow for 32*32 dataset? Since only about 72% acc(1.5 version) would be obtained after 400 epoch using the same training paras as this repo.

        self.network_config = [
            g_name('data/bn', nn.BatchNorm2d(3)),
            slim.conv_bn_relu('stage1/conv', 3, in_channels, 1, 1, 1),   #3,2,1->1,1,1
            #g_name('stage1/pool', nn.MaxPool2d(3, 2, 0, ceil_mode=True)),  #removed
            (width_config[0], 2, 1, 4, 'b'),
            (width_config[1], 2, 1, 8, 'b'), # x16
            (width_config[2], 2, 1, 4, 'b'), # x32
            slim.conv_bn_relu('conv5', width_config[2], width_config[3], 1),
            g_name('pool', nn.AvgPool2d(4, 1)), #7->4
            g_name('fc', nn.Conv2d(width_config[3], self.num_classes, 1)),
        ]

Thanks very much for your help!

**Comment by miaow1988** [comment]
The problem might be 1x1 conv layers you used:
```
slim.conv_bn_relu('stage1/conv', 3, in_channels, 1, 1, 1),   #3,2,1->1,1,1
```
You can try 3x3_s1 instead as following:
```
slim.conv_bn_relu('stage1/conv', 3, in_channels, 3),
```

**Comment by cvtower** [comment]
Thanks very much for the answer!

However, with this patch, only ~72% acc will be obtained again after 400 epoch for cifar100:
x1.5(x1.0 turned to be worse)+batch size 512, and all training paras keep the same with this repo, data augmentation module keeps the same with mobilenetv2, weird acc value but this is it.

**Comment by miaow1988** [comment]
Something you may try.
1. Double check your train/val data.
2. Use ImageNet pretrained models.
3. Change network structure, e.g., do not use down sample in first two stage.
